src/main/java/network/brightspots/rcv/BaseCvrReader.java

```diff
@@ -39,6 +39,11 @@ abstract class BaseCvrReader {
   abstract void readCastVoteRecords(List<CastVoteRecord> castVoteRecords)
       throws CastVoteRecord.CvrParseException, IOException;
 
+  // Individual contests may have a different value than what the config allows.
```

Comment:
Curious what an example of this would be?

Reply:
All dominion files have a different value. The Dominion XML specifies one number, and it doesn't necessarily correspond to the config value.

Reply:
Weird, ok!

Reply:
Should we disable the config setting when all of the CVR sources are Dominion? And/or should we clarify that this setting is ignored for Dominion sources?

Reply:
Sounds reasonable to describe this somewhere visible if even one of the configs is Dominion. Since this is an issue with the current code too, reasonable to push to another Issue?